packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -395,6 +396,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
             inlineImageOptions={inlineImageOptions}
             numberOfChatMessagesToReload={defaultNumberOfChatMessagesToReload}
             styles={messageThreadStyles}
+            strings={{ sunday: '134', editBoxSubmitButton: '456', boldTooltip: '123' }}
```

Comment:
The RichText strings should be optional so if RTE is not enabled, Contoso don't have to override it.